src/perturbopy/postproc/calc_modes/spectral_cumulant.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import h5py as h5
+import matplotlib.pyplot as plt
+from perturbopy.postproc.calc_modes.calc_mode import CalcMode
+
+
+class SpectralCumulant(CalcMode):
+    """
+    A class to handle and analyze spectral functions computed using the cumulant expansion approach.
+
+    Attributes:
+    - T (numpy.ndarray): Array of temperatures corresponding to the spectral data.
+    - w (numpy.ndarray): Array of energy values (ω) in electron volts (eV).
+    - dw (float): Energy step size for the ω grid in eV.
```

Comment:
Please use more explicit names than `T, w, dw`, instead, use something like `temp_array`, `omega_array`, etc., the exact names are up to you of course.